guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+
+## Signing out
+
+Ensuring users are signed out (manually or via session time-outs) is important. The account may have payment allowances, or other problematic permissions, that could otherwise be abused by third-parties.
+
+## Making payments
+
+A payment is typically initiated by your service when the user wants to take an action. Show the most appropriate option right away based on how the user has previously interacted with their wallet (direct or bridged, [see above]({{ '#wallet-service-communication' | relative_url }})).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/payment-request.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/payment-request@2x.png"
+   alt-text = "Browser window with a modal for confirming a payment of 100 satoshi"
+   width = 800
+   height = 411
+%}
+
+Allowances allow for automatic approval of payments by wallets up to certain amounts. This reduces friction for users that frequently transact. With an allowance, the modal window in the visualization above would not be needed. Ensure your interface provides user feedback when payments are confirmed.
+
+An alternative to reducing friction for frequent payments is to let users deposit satoshi into their accounts (and later withdraw them). This puts the service into a custodial position, for the benefit of smoother payments (since they only involve database updates and not actual lightning payments).
+
+## Connecting multiple wallets
+
+Users may migrate wallets and want to link their existing account to a new wallet. They may also want to connect multiple wallets to the same account. This could be for convenience, like signing in with both a desktop and a mobile wallet. Or it could be for security, like logging in with one wallet, but making payments with another. It may also be necessary on order to accomodate different implementations and lack of [interoperability]({{ '/guide/designing-products/interoperability/' | relative_url }})).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/connected-wallets.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/connected-wallets@2x.png"
+   alt-text = "Browser window showing an account screen with a list of linked wallets"
+   width = 800
+   height = 411
+%}
+
+## Recovery
+
+If a user loses their wallet, they also lose their access to your service. Ensure you have a recovery mechanism in place. This could be a single-use recovery code (as is a common pattern with [multi-factor authentication](https://en.wikipedia.org/wiki/Multi-factor_authentication)), or a traditional email & password setup that you can also use for other communication with the user.
```

Comment:
```suggestion
If a user loses their wallet, they also lose their access to your service. Ensure you have a recovery mechanism in place. This could be a single-use recovery code (as is a common pattern with [multi-factor authentication](https://en.wikipedia.org/wiki/Multi-factor_authentication)) or a traditional email & password setup that you can also use for other communication with the user.
```